webdriver/bidi/session.json

```diff
@@ -79,7 +77,11 @@
               "firefox": {
                 "version_added": "92",
                 "partial_implementation": true,
-                "notes": "Basic support (see [bug 1719290](https://bugzil.la/1719290))"
+                "notes": [
+                  "Starting with Firefox 116 the command supports capability matching.",
+                  "",
+                  "Multiple session creation is not currently supported by this command."
```

Comment:
@whimboo If there is a bug for this, it would be useful to link here, to make it easier to update this data when it gets resolved.

Reply:
Sure, but it's a meta bug. I've added it for now. Feel free to remove if it is not what you want.